hrm-service/src/permissions/middlewares.js

```diff
@@ -13,8 +12,9 @@ const canEditCase = asyncHandler(async (req, res, next) => {
   if (!req.isAuthorized()) {
     const { accountSid, body, user, can } = req;
     const { id } = req.params;
-    const caseObj = await CaseController.getCase(id, accountSid);
-    const actions = Actions.getActions(caseObj.dataValues, body);
+    const caseObj = await getById(id, accountSid);
+    const caseModel = new Case(caseObj);
```

Comment:
The permissions system still needs Sequelize Case models (but no actual Sequelize DB Access as far as I know). I decided to leave it as tech debt for the time being